mindmeld/server.py

```diff
@@ -85,17 +93,24 @@ def parse():
             if request_json is None:
                 msg = "Invalid Content-Type: Only 'application/json' is supported."
                 raise BadMindMeldRequestError(msg, status_code=415)
-
-            safe_request = {}
-            for key in ["text", "params", "context", "frame", "history", "verbose"]:
-                if key in request_json:
-                    safe_request[key] = request_json[key]
-            response = self._app_manager.parse(**safe_request)
-            # add request id to response
-            # use the passed in id if any
-            request_id = request_json.get("request_id", str(uuid.uuid4()))
-            response.request_id = request_id
-            return jsonify(DialogueResponder.to_json(response))
+            try:
+                type_validated_request_json = self._request_schema.load(request_json)
+                safe_request = {}
+                for key in ["text", "params", "context", "frame", "history", "verbose"]:
+                    if key in type_validated_request_json:
+                        safe_request[key] = type_validated_request_json[key]
```

Comment:
Do we still need this? Can we just pass `type_validated_request_json` in to app_manager.parse?

Reply:
great point, removed.